src/chapters/typescript/compiling-typescript.rst

```diff
@@ -0,0 +1,64 @@
+Compiling TypeScript
+====================
+
+To work with TypeScript in our Node environment, we'll need to install the TypeScript
+module. In this section, you'll install the compiler and practice compiling and 
+running your TypeScript programs all from the terminal.
+
+To install Typescript, in your terminal, type
+
+.. sourcecode:: bash
+
+   $ npm install -g typescript
+
+Once you have TypeScript installed, you'll be able to run 
+
+.. sourcecode:: bash
+
+   $ tsc -v
+
+``tsc`` is is the TypeScript compiler and ``-v`` is the version flag to show us that
+we have TypeScript and it's compiler installed. 
+
+In true tutorial form, let's write a quick ``Hello, World`` program so we can see this 
+compiling in action.
+
+In your code editor, create a new file called ``hello_world.ts``. In it, initialize a 
+variable called ``message`` with the string value ``'Hello, World'``. Print the message.
+
+.. sourcecode:: ts
+   :linenos:
+
+    let message: string = 'Hello, World';
+    console.log(message);
+
+
+Return to the terminal in the directory that houses your new file and compile your 
```

Comment:
The phrasing here is a little awkward.  How about something like: 
"In the terminal, navigate into the directory that houses your ``hello_world.ts`` file. Compile the file with the following command:"